setup.py

```diff
@@ -36,6 +36,7 @@ def run(self):
     ],
```

Comment:
Can you merge main? I think this setup.py is likely out-of-date now that the other reader is already in.